<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="tableView"/>
      <widget class="QGraphicsView" name="graphicsView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuDataBase">
     <property name="title">
      <string>DataBase</string>
     </property>
     <addaction name="actionCreate_Connection"/>
     <addaction name="actionCreate_table_DB"/>
     <addaction name="actionWrite_to_DB"/>
     <addaction name="actionRead_from_DB"/>
     <addaction name="actionQuery_Model"/>
     <addaction name="actionTable_Model"/>
    </widget>
    <addaction name="actionRectangle"/>
    <addaction name="actionColor"/>
    <addaction name="actionSettings"/>
    <addaction name="actionClear"/>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
    <addaction name="menuDataBase"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRectangle"/>
   <addaction name="actionColor"/>
   <addaction name="actionSettings"/>
   <addaction name="actionClear"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/MyImages/Images/Rect.png</normaloff>:/MyImages/Images/Rect.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/MyImages/Images/changecolor.png</normaloff>:/MyImages/Images/changecolor.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/MyImages/Images/settings-256.png</normaloff>:/MyImages/Images/settings-256.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/MyImages/Images/clearicon.png</normaloff>:/MyImages/Images/clearicon.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/MyImages/Images/saveicon.png</normaloff>:/MyImages/Images/saveicon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/MyImages/Images/openicon.png</normaloff>:/MyImages/Images/openicon.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/MyImages/Images/iconExit.ico</normaloff>:/MyImages/Images/iconExit.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCreate_Connection">
   <property name="text">
    <string>Create Connection</string>
   </property>
  </action>
  <action name="actionCreate_table_DB">
   <property name="text">
    <string>Create table DB</string>
   </property>
  </action>
  <action name="actionWrite_to_DB">
   <property name="text">
    <string>Write to DB</string>
   </property>
  </action>
  <action name="actionRead_from_DB">
   <property name="text">
    <string>Read from DB</string>
   </property>
  </action>
  <action name="actionQuery_Model">
   <property name="text">
    <string>Query Model</string>
   </property>
  </action>
  <action name="actionTable_Model">
   <property name="text">
    <string>Table Model</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myimages.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
